packages/react-components/lib/reports/user-report/user-login-failure-report-table.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import MaterialTable from 'material-table';
+import { Typography } from '@material-ui/core';
+import moment from 'moment';
+import { materialTableIcons } from '../../material-table-icons';
+import { DefaultLogTableProps } from '../default-report-interface';
+
+export type UserLoginFailureRowsType = {
+  created: string; //date
+  ip_address: string;
+  payload: string | unknown;
+  client_id: string;
+  username: string;
+  error: string;
+}[];
+
+export interface UserLoginFailureReportTable extends DefaultLogTableProps {
+  rows: UserLoginFailureRowsType | [];
```

Comment:
Why do we need `[]`? I thought `UserLoginFailureRowsType` already allows an empty array.

Reply:
Yeah, you're right, we don't. Thanks for pointing this!